EasyClangComplete.py

```diff
@@ -144,6 +166,9 @@ def on_plugin_loaded(self):
         self.on_settings_changed()
         # init view config manager
         self.view_config_manager = ViewConfigManager()
+        global view_config_manager
```

Comment:
I don't really get the reason behind using a global view config manager. There is one that is specific to the current class. Why introduce a new one?

Reply:
because this config manager is used in both 
class easy_clang_goto_declarationCommand
and 
class EasyClangComplete

is there any other way to share this config manager between classes?
Is it possible to use class EasyClangComplete to implement goto_declaration command?
I tried to use on_text_command() event handler and match command by name. But as for me, it is not very good idea, because on_text_command() will receive any possible commands so it can be performance degradation



P.S. I am not faimiliar with sublimetext plugins. This is my first attempt to deal with it's plugins.

Reply:
I have moved the config manager to be static. It makes sense and we can avoid keeping the different managers in sync.